modules/installation-three-node-cluster-cloud-provider.adoc

```diff
@@ -38,16 +42,26 @@ Because application workloads run from control plane nodes, additional subscript
 
 .Procedure
 
+ifdef::nutanix[]
+* Set the number of compute replicas to `0` in your `install-config.yaml` file, as shown in the following `compute` stanza:
+endif::nutanix[]
+
+ifndef::nutanix[]
 . Set the number of compute replicas to `0` in your `install-config.yaml` file, as shown in the following `compute` stanza:
+endif::nutanix[]
 +
+.Example `install-config.yaml` file for a three-node cluster
 [source,yaml]
 ----
+apiVersion: v1
+baseDomain: example.com
 compute:
 - name: worker
   platform: {}
   replicas: 0
+# ...
 ----
-ifndef::vsphere,vmc[]
+ifndef::vsphere,vmc,nutanix[]
```

Comment:
You might get a merge conflict here if https://github.com/openshift/openshift-docs/pull/60614 gets merged before your PR.

Reply:
Yeah. Probably, but easy enough to fix. Thanks.